packages/pds-ember/app/styles/pds/components/popup/__config.scss

```diff
@@ -0,0 +1,130 @@
+///-------------------------------------------------------------------------///
+/// WARNING: Private Sass module (definitions may change at ANY time)
+///-------------------------------------------------------------------------///
+$_module: "PDS.Components.Popup";
+/* --- [debug] CONFIG: #{$_module} --- */
+@use "sass:map";
+@use "../../theme";
+@use "../../tokens/color";
+@use "../../tokens/font" as Font;
+@use "../../tokens/scale";
+@use "../_config" as Component;
+@use "../button/_config" as Button;
+@use "../icon/_config" as Icon;
+@use "../banner/_config" as Banner;
+@use "../../core/typography/_config" as Typography;
+
+$__size14: 14px; // FIXME: replace with PDS.Tokens.Scale value
+$__icon-color: var(--pds-popup-icon-color, inherit);
+$__header-color: var(--pds-popup-header-color, inherit);
+
+$backgroundColor: var(--pds-popup-backgroundColor, transparent);
+$borderColor: var(--pds-popup-borderColor, transparent);
+$borderRadius: theme.$borderRadius;
+$color: var(--pds-popup-color, inherit);
+$columnGap: theme.$size--2xs; // 0.25rem (4px)
+$fontSize: theme.$popup-fontSize;
+$paddingBlock: $__size14;
+$paddingInline: theme.$size--sm; // 0.75rem / 12px (scale.$sm--1 / 0.75rem / 12px)
+$rowGap: 0; // FIXME: use theme.$size--xs; // 0.5rem (8px)
+
+/// ===== VARIANT STYLES ===== ///
+$ERROR: Banner.$ERROR;
+$INFO: Banner.$INFO;
+$SUCCESS: Banner.$SUCCESS;
+$WARNING: Banner.$WARNING;
+
+@mixin theme($styles) {
+  /* [debug] #{$_module}@theme */
+  @include Component.theme(popup, $styles);
+}
+
+@mixin layout {
+  /* [debug] #{$_module}@layout */
+  padding: $paddingBlock $paddingInline;
+
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  grid-template-areas:
+    'icon header close'
```

Comment:
Is there a way to apply vertical centering for the top row? The simple variant is visually off-center because of this.

![CleanShot 2020-12-01 at 12 27 10](https://user-images.githubusercontent.com/39469/100781403-9753d580-33d0-11eb-9480-9d333e280932.png)

vs (this is centered vertically): 

![CleanShot 2020-12-01 at 12 27 01](https://user-images.githubusercontent.com/39469/100781417-9f137a00-33d0-11eb-87a6-c472153ede0f.png)

Reply:
Design wishes for the title to be able to wrap if it can't fit on one line, because the title may contain important information and should never be truncated.  This requires the title to have a `line-height` that is greater than 1 to ensure accessible whitespace between lines of text.  This extra whitespace makes vertical centering of icons unpredictable when compared to text, depending on the loaded font.  If the title were guaranteed to always be 1 line of text, we might be able to get away with using `align-items: center`, but the positioning of the icon and "x" would shift downward when a multi-line title is present.

Reply:
Had a chat with Design. We'll make a tweak to get closer to center, but it won't be perfect.

Reply:
@meirish : tweaks have been made to better "center" the icon and title